subsys/net/lib/lwm2m/lwm2m_rw_oma_tlv.c

```diff
@@ -903,6 +903,53 @@ static int write_tlv_resource(struct lwm2m_message *msg, struct oma_tlv *tlv)
 	return 0;
 }
 
+static int lwm2m_multi_resource_tlv_parse(struct lwm2m_message *msg,
+					  struct oma_tlv *multi_resource_tlv)
+{
+	struct oma_tlv tlv_resource_instance;
+	int len2;
+	int pos = 0;
+	int ret;
+
+	if (msg->in.block_ctx) {
+		msg->in.block_ctx->res_id = multi_resource_tlv->id;
+	}
+
+	if (multi_resource_tlv->length == 0U) {
+		/* Mo nested data Create path level 3  */
+		return write_tlv_resource(msg, multi_resource_tlv);
+	}
+
+	while (pos < multi_resource_tlv->length &&
+	       (len2 = oma_tlv_get(&tlv_resource_instance, &msg->in, true))) {
+		if (tlv_resource_instance.type != OMA_TLV_TYPE_RESOURCE_INSTANCE) {
+			/* Skip Unsupported TLV type */
```

Comment:
Unnecessary comment. The if-clause already tells what we are looking for, and LOG output is telling that it is not supported.

Reply:
ok